aiida/tools/importexport/dbimport/backends/sqla.py

```diff
@@ -0,0 +1,681 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'm going to mostly skip this file (I guess most comments apply both to the django/sqla version)

Reply:
yep 👍